lib/zendesk_apps_support.rb

```diff
@@ -10,6 +10,7 @@ module ZendeskAppsSupport
   autoload :Package,                'zendesk_apps_support/package'
   autoload :Installed,              'zendesk_apps_support/installed'
   autoload :Installation,           'zendesk_apps_support/installation'
+  autoload :Finders,                'zendesk_apps_support/finders'
```

Comment:
curious, does this do anything given that the code always `extend ZendeskAppsSupport::Finders`?